### DISCUSSION TITLE
How to get current state of the audio player ?

### DISCUSSION THREAD

**Discussion by Hitesh0413** [bluefireteam/audioplayers#1286] [General]
I there any method to get the current state? The onStateChange method is present in the library I will only invoke it when there is a state change .. but I want to get the state on demand.

**Comment by Gustl22** [comment]
Just call `audioplayer.state`. See: https://github.com/bluefireteam/audioplayers/blob/4aaee70771092b2dc52375584dc61a5990abf08d/packages/audioplayers/example/lib/tabs/streams.dart#L58
